wireapp/wire-ios-mono
# feat: mls reject over self conversation - WPB-608

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-608" title="WPB-608" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-608</a>  [iOS] 5.1.4 MLS REJECT over self-conversation
  </td></table>
  <br />
 






































# What's new in this PR?

### Issues

This PR implements sending reject message in a group call to all self clients via MLS.

Basically when receiving a call, and I reject it, all my other clients (web,...) stop ringing.

### Dependencies (Optional)

https://github.com/wireapp/wire-ios-mono/pull/275

### Testing

#### Test Coverage (Optional)


#### How to Test

Manually:
1. Create a team on Anta with a MLS group
2. Have user1 connected via web and iPhone
3. Have user2 called on the group
4. user1 rejects call on iPhone
-> user1 on web should not ring anymore.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
